CSC8503/SuspicionSystem/GlobalSuspicionMetre.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#include "SuspicionMetre.h"
+#include <vector>
+
+const int DT_UNTIL_GlOBAL_RECOVERY = 8;
+class GlobalSuspicionMetre :
+    public SuspicionMetre
+{
+public:
+    const enum instantGlobalSusCause
+    {
+        allarmTriggered, flagCaptured
+    };
+
+    const enum activeGlobalSusCause
```

Comment:
Instant vs active is a bit confusing - I think what you mean is 'instant' happens in one instant, and 'active' is over a time period?

Perhaps Single and Continuous or Repeating?